src/status_im/contexts/wallet/events/collectibles.cljs

```diff
@@ -66,8 +65,9 @@
          data-type           (collectible-data-types :header)
          fetch-criteria      {:fetch-type            (fetch-type :fetch-if-not-cached)
                               :max-cache-age-seconds max-cache-age-seconds}
+         chain-ids           (rf/sub [:wallet/networks-chain-id-by-mode])
```

Comment:
@mmilad75 - We should not use a subscription inside a re-frame event as it may cause a memory leak. We can create helper methods to return the chain-ids just like we have for single chain-id.

Reply:
Done.